eslint.config.cjs

```diff
@@ -1,111 +1,125 @@
-          'newlines-between': 'always',
-          alphabetize: { order: 'asc', caseInsensitive: true },
-        },
-      ],
-      'import/prefer-default-export': 'off',
-
-      // Calidad general (no de formato)
       eqeqeq: ['error', 'always', { null: 'ignore' }],
       'no-var': 'error',
       'prefer-const': ['error', { destructuring: 'all' }],
-      'no-unused-vars': 'off', // lo maneja TS en archivos TS
+      'no-unused-vars': 'warn',
     },
   },
-  ...ts.configs['flat/recommended'],
+
+  // TypeScript files
   {
     files: ['**/*.{ts,tsx}'],
     languageOptions: {
       parser: tsParser,
-      parserOptions: { ecmaVersion: 'latest', sourceType: 'module' },
-    },
-    plugins: { '@typescript-eslint': ts, import: importPlugin, prettier: prettierPlugin },
-    settings: {
-      'import/resolver': {
-        typescript: { alwaysTryTypes: true, project: ['./tsconfig.json'] },
-        node: true,
+      parserOptions: { 
+        ecmaVersion: 'latest', 
+        sourceType: 'module',
+        ecmaFeatures: {
+          jsx: true
+        }
       },
+      globals: {
+        React: 'readonly',
+        window: 'readonly',
+        document: 'readonly',
+        global: 'readonly',
+        HTMLElement: 'readonly',
+      }
+    },
+    plugins: { 
+      '@typescript-eslint': ts, 
+      prettier: prettierPlugin,
+      'react-hooks': reactHooks,
+      'react-refresh': reactRefresh,
     },
     rules: {
       'prettier/prettier': 'error',
-      'import/no-unresolved': 'error',
       '@typescript-eslint/quotes': [
         'error',
         'single',
-        { avoidEscape: true, allowTemplateLiterals: false },
-      ],
-      'import/no-duplicates': 'error',
-      'import/extensions': ['error', 'ignorePackages', { ts: 'never', tsx: 'never' }],
-      'no-console': 'error',
-      'import/order': [
-        'error',
-        {
-          groups: [
-            'builtin',
-            'external',
-            'internal',
-            'parent',
-            'sibling',
-            'index',
-            'object',
-            'type',
-          ],
-          'newlines-between': 'always',
-          alphabetize: { order: 'asc', caseInsensitive: true },
+        { 
+          avoidEscape: true, 
+          allowTemplateLiterals: false 
         },
       ],
-
-      // Preferir reglas de TS sobre las base
+      'no-console': 'warn',
       'no-unused-vars': 'off',
-      // '@typescript-eslint/no-shadow': 'error', // activar si lo necesitás
-      // Ajustes comunes de Airbnb (opcionales)
-      // 'no-plusplus': 'off',
-      // 'no-underscore-dangle': 'off',
-      // 'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['acc', 'e', 'ctx', 'req', 'res', 'state'] }],
+      '@typescript-eslint/no-unused-vars': [
+        'warn',
+        { 
+          argsIgnorePattern: '^_',
+          varsIgnorePattern: '^_',
+        }
+      ],
+      '@typescript-eslint/no-explicit-any': 'warn',
```

Comment:
Rule `@typescript-eslint/no-explicit-any` is configured as 'warn'. Project rules require NEVER using `any` (the project instructions say to avoid `any` entirely). Allowing or warning for `any` contradicts that guideline.

**Sugerencia:**
```typescript
Make this rule stricter to enforce the project guideline. Set it to 'error' (or at least 'off' is not acceptable if policy is to never use any):

'@typescript-eslint/no-explicit-any': 'error',

If there are legitimate cases for `any`, add a narrow inline exception with a code comment and a link to justification instead of loosening the rule globally.
```